force-app/main/default/classes/AttendanceController.cls

```diff
@@ -39,4 +39,21 @@ public with sharing class AttendanceController {
             throw Util.getAuraHandledException(ex);
         }
     }
+
+    @AuraEnabled(cacheable=true)
+    public static Map<String, List<String>> getServiceSessionStatusBuckets(
```

Comment:
What do you think about defining the bucketNames, objectType, and bucketedField as properties of the AttendanceController instead of passing them in as arguments? The name of this method suggests it only returns buckets for the status field on ServiceSession, but the way it's coded the caller can actually get bucket info for any field an object. 

If the bucketNames, objectType, and bucketedField are defined by the AttendanceController class instead of passed in from the javascript, I think the interface is a little big cleaner.

Reply:
(Or just defining those values in the method itself probably works too)